src/actors/sheet-parts/background-tab.hbs

```diff
@@ -1,107 +1,109 @@
-    </label>
-    {{else if data.data.background.tribe}}
-      <b>{{localize 'RQG.Tribe'}}</b> {{data.data.background.tribe}}
-    {{/if}}
-  </div>
-
-  <div class="nowrap">
-    {{#if isGM}}
-    <label>{{localize 'RQG.Clan'}}
-      <input type="text" name="data.background.clan" value="{{data.data.background.clan}}" size="30">
-    </label>
-    {{else if data.data.background.clan}}
-      <b>{{localize 'RQG.Clan'}}</b> {{data.data.background.clan}}
-    {{/if}}
-  </div>
-
-</fieldset>
-
-<fieldset>
-  <legend>{{localize 'RQG.Circumstance'}}</legend>
+      {{else if data.data.background.species}}
+      <b>{{localize 'RQG.Actor.Background.Species'}}</b> {{data.data.background.species}}
+      {{/if}}
+    </div>
+  </fieldset>
+
+  <fieldset>
+    <legend>{{localize 'RQG.Actor.Background.Origin'}}</legend>
+
+    <div class="nowrap">
+      {{#if isGM}}
+      <label>{{localize 'RQG.Actor.Background.Homeland.Heading'}}
+        <select name="data.background.homeland">
+          {{#select data.data.background.homeland}} {{#each homelands}}
+          <option value="{{this}}">{{localize (concat 'RQG.Actor.Background.Homeland.' this)}}</option>
+          {{/each}} {{/select}}
+        </select>
+      </label>
+      {{else if data.data.background.homeland}}
+      <b>{{localize 'RQG.Actor.Background.Homeland.Heading'}}</b> {{localize (concat 'RQG.Actor.Background.Homeland.'
+      data.data.background.homeland)}}
+      {{/if}}
+    </div>
+
+    <div class="nowrap">
+      {{#if isGM}}
+      <label>{{localize 'RQG.Actor.Background.TownHomeArea'}}
+        <input type="text" name="data.background.town" value="{{data.data.background.town}}" size="30">
+      </label>
+      {{else if data.data.background.town}}
+      <b>{{localize 'RQG.Actor.Background.TownHomeArea'}}</b> {{data.data.background.town}}
+      {{/if}}
+    </div>
+
+    <div class="nowrap">
+      {{#if isGM}}
+      <label>{{localize 'RQG.Actor.Background.Tribe'}}
+        <input type="text" name="data.background.tribe" value="{{data.data.background.tribe}}" size="30">
+      </label>
+      {{else if data.data.background.tribe}}
+      <b>{{localize 'RQG.Actor.Background.Tribe'}}</b> {{data.data.background.tribe}}
+      {{/if}}
+    </div>
+
+    <div class="nowrap">
+      {{#if isGM}}
+      <label>{{localize 'RQG.Actor.Background.Clan'}}
+        <input type="text" name="data.background.clan" value="{{data.data.background.clan}}" size="30">
+      </label>
+      {{else if data.data.background.clan}}
+      <b>{{localize 'RQG.Actor.Background.Clan'}}</b> {{data.data.background.clan}}
+      {{/if}}
+    </div>
+
+  </fieldset>
+
+  <fieldset>
+    <legend>{{localize 'RQG.Actor.Background.Circumstance'}}</legend>
 
     <div class="nowrap">
       {{#if isGM}}
       <span data-reputation-roll>
-        <img src="{{defaultItemIconSrc "Reputation"}}" class="item">
+        <img src="{{defaultItemIconSrc " Reputation"}}" class="item">
```

Comment:
Remove extra space
```suggestion
        <img src="{{defaultItemIconSrc "Reputation"}}" class="item">
```

Reply:
AAAARGH!!!!!!  The "Format Document" does that.  I have to remember to either not format the document or remember to fix it.  Pain in the butt.

Reply:
Done